waspc/tools/make_binary_package.sh

```diff
@@ -9,15 +13,25 @@
 DST=$PWD/${1:-wasp.tar.gz}
 
 TMP_DIR="$(mktemp -d 2> /dev/null || mktemp -d -t wasp-bin-package)"
+CABAL_PROJECT_ROOT_PATH="$(cabal list-bin wasp-cli | sed s/\\/dist-newstyle.*//)"
 
 WASP_BINARY_PATH="$(cabal list-bin wasp-cli)"
 cp "$WASP_BINARY_PATH" "$TMP_DIR/wasp-bin"
 
-CABAL_PROJECT_ROOT_PATH="$(cabal list-bin wasp-cli | sed s/\\/dist-newstyle.*//)"
-cp -R "$CABAL_PROJECT_ROOT_PATH/data" "$TMP_DIR/data"
+CABAL_DATA_FILE_PATHS=$(
+  # First we list all the files declared in the .cabal file
+  # Then we only take the ones in the "data" directory
+  cabal sdist --list-only | grep "^./data/"
```

Comment:
Not a big deal, but the dot is interpreted as "any character" not a literal dot. We should escape it.